lib/Backends/OpenCL/OpenCLDeviceManager.h

```diff
@@ -119,6 +119,15 @@ class OpenCLDeviceManager : public QueueBackedDeviceManager {
   void runFunctionImpl(runtime::RunIdentifierTy id, std::string functionName,
                        std::unique_ptr<Context> ctx, ResultCBTy cb) override;
 };
+
+class OpenCLDeviceConfig : public DeviceConfig {
+public:
+  OpenCLDeviceConfig() : DeviceConfig(BackendKind::OpenCL) {}
+  unsigned platformId;
```

Comment:
This class has only public members. Make it a struct or make the data members private (and do not forget the underscores at the end) and provide accessors.